client/lib/promote-post/index.ts

```diff
@@ -205,6 +213,12 @@ export async function showDSP(
 	} );
 }
 
+export async function cleanupDSP() {
```

Comment:
Is this needed any more?

Reply:
Yes, we still need to do some cleanups in the widget.